README.md

```diff
@@ -104,6 +114,32 @@ Host your own Chocolatey package repository
 
 #### Parameters
 
+##### `chocolatey_server_app_pool_name`
+Set apppool name used by the chocolatey.server website. Defaults to
+'chocolatey.server'.
+
+##### `compatibility_mode`
+Compatibility mode was added to allow this module to be backwards
+compatible with older implementations. Defaults to 'true'.
+
+##### `disable_default_website`
+The default website is stopped to prevent conflicts with the
+chocolatey server website. An alternate port can be defined and
+this option can be set to disabled so it doesn't modify the default
+website. Defaults to 'true'.
+
+##### `default_website_path`
+The default website path is needed in order to modify the default
+website. Defaults to 'C:\inetpub\wwwroot'.
```

Comment:
I'm not sure this is necessary with WindowsFeatures module - I'm hesitant to convert to Puppet+DSC as the native option is there. I will explain why native option is better in my review comments at the end.